### PR TITLE
Dcc order splitter

### PR DESCRIPTION
**Pull request by koriit-kontakt**

### CODE REVIEW

**Comment by koriit-kontakt** [comment]
Added dcc_device_diff for unclaims and dcc_move_order for moving orders between environments

File: dcc_diff_device.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        "irConfig": {
+            "irDetectionInterval": device.get("irReceiver", {}).get("wakeUpInterval"),
+            "irDetectionDuration": device.get("irReceiver", {}).get("maxListenInterval"),
+            "irDetectionSleep": device.get("irReceiver", {}).get("disableOnNoMotion"),
+        },
+        "bleScanConfig": {
+            "bleScanDuration": device["fingerprinting"]["duration"],
+            "bleScanInterval": device["fingerprinting"]["interval"],
+        },
+        "occupancyConfig": {
```
Comment (adrianz) [inline dcc_diff_device.py:97]: It already misses `occupancy_sleep_interval_s`

**Review by adrianz — APPROVED** [review]
From a technical POV it looks fine, but I think it will be another tool that we will have to constantly maintain, because it hardcodes all the data structures from IM API and DCC

File: dcc_move_order.py [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+
+def import_table(cursor, table, filename):
+    with open(filename, "r") as f:
+        with cursor.copy(f"COPY {table} FROM STDIN") as copy:
+            copy.write(f.read())
+
+
+def claim_order(cursor, order, cloud_id):
+    cursor.execute(
+        "UPDATE \"order\" SET claimed_date = current_timestamp, cloud_id = %(cloud_id)s, company_id = %(company_id)s WHERE id = %(id)s",
```
Comment (vpanovk) [inline dcc_move_order.py:282]: I'm really afraid to modify database in these scripts. I hope you know what you're doing here ;)
Anyway, the where `id=?` clause seems sensible.

**Review by vpanovk — APPROVED** [review]

File: dcc_diff_device.py (hunk 1 above)
Comment (koriit-kontakt) [reply dcc_diff_device.py:97]: Yes, it's a problem. Ultimately, this tool just helps us move the configuration changes from DM to DCC before unclaim. This should eventually be implemented in DM itself. Until then no other choice... If something is missed, then that's it, and all...

**Pull request merged**